src/tif_model_iterator/tif_kernel_iterator.py

```diff
@@ -360,23 +301,12 @@ def _write_part_to_file(
         @param step: the step currently being written to file
         """
         print("Dissolving and writing to file")
-        start = timer()
+
         output_file_name = self.output_file_name_generator.generate_part_output_path(
             step
         )
 
-        gdf = gdf.dissolve(by="label")
-        print("Dissolving finished in: " + str(timer() - start) + " second(s)")
-
-        start = timer()
-
-        if ".geojson" in output_file_name:
-            print("Writing part to geojson")
-            gdf.to_file(output_file_name, driver="GeoJSON")
-        else:
-            gdf.to_file(output_file_name)
-
-        print("Writing finished in: " + str(timer() - start) + " second(s)")
+        dissolve_gpd_output(gdf, output_file_name)
```

Comment:
Do I understand correctly that this makes it into a dataframe with a row per label with a MultiPolygon as geometry for all the polygons?

Reply:
Okay dokay!